internal/cli/cmd/dac/setup/setup.go

```diff
@@ -128,23 +169,42 @@ func (o *option) Execute() error {
 
 	// Install dependencies
 	if o.language == cueLanguage {
-		if err := cue.InstallCueDepsFromSources(cueSchemasPath, o.version); err != nil {
-			return fmt.Errorf("error installing the Perses CUE schemas as dependencies: %v", err)
+		// Check cue.mod exists
+		if _, err := os.Stat("cue.mod"); os.IsNotExist(err) {
+			return fmt.Errorf("unable to find the 'cue.mod' dir. Please run 'cue mod init'")
+		} else if err != nil {
+			return err
+		}
+		// Resolve the sdk dependency (= write an example file
+		// and run 'cue mod tidy')
+		exampleFilePath := "example.cue"
+		if err := os.WriteFile(exampleFilePath, []byte(exampleCUEDac), 0600); err != nil {
+			return fmt.Errorf("failed to create example.cue: %w", err)
+		}
+		logrus.Debugf("example.cue file created successfully")
+		cmd := exec.Command("cue", "mod", "tidy")
```

Comment:
I hope we won't need anymore to use a token to get the dependencies

Reply:
Well yes you have to be logged in everytime you need to resolve dependencies.. AFAIS when working with dac it's not that annoying, the authentication persist for quite a long time.